### DISCUSSION TITLE
Building defences, resilience and unfinished buildings

### DISCUSSION THREAD

**Discussion by Indy2222** [DigitalExtinction/Game#149] [Ideas]
The goal of this discussion is to evaluate and design damage system.

* Resilience of an unfinished building is dis-proportionally smaller to a finished building. This should be in line with the overall damage system rather than a special case.

**Comment by JackCrumpLeys** [comment]
One way to approach this is to give unfinished buildings a higher chance of sustaining critical damage or collapsing under stress. For example, an unfinished building may be more likely to collapse if hit by a large explosion, while a finished building may sustain more damage but remain standing.

Does damage decrease building progress?
